Launcher/Classes/ViewControllers/SettingsViewController.swift

```diff
@@ -151,13 +165,13 @@ class SettingsViewController: NSViewController {
         }
     }
     
-    func fillDictionary(with value: String, for key: String, on fieldName: String) -> [String : Any] {
+    func fillDictionary(with value: String, for key: String) -> [String : Any] {
```

Comment:
Looking at this, I have no idea what this function is doing / trying to accomplish. Can you explain what it is you're expecting here?

Reply:
It fills the `Dictionary` with 'some' values for 'some' key. Will rename the method